endpoints/openrtb2/video_auction.go

```diff
@@ -366,6 +366,9 @@ func buildVideoResponse(bidresponse *openrtb.BidResponse, podErrors []PodError)
 			if err := json.Unmarshal(bid.Ext, &tempRespBidExt); err != nil {
 				return nil, err
 			}
+			if tempRespBidExt.Prebid.Targeting["hb_uuid"] == "" {
```

Comment:
Hey @Kalypsonika! I just noticed that we have constants defined for different Targeting Keys in `openrtb_ext/bid.go`. I'd suggest we probably use that here instead of the hardcoded string.
 
Also, I understand that we use hard coded strings for these in most of the places in the code and I think we should probably refactor that. Maybe, I will put up a PR for the same some time in the future but that shouldn't stop us from changing it here now. What do you think?

Reply:
@mansinahar Hi Mansi! I totally agree, we don't want to see any hardcoded strings like this. I'll change all of them in the video endpoint.

Reply:
Thank you so much. Appreciate it!